src/pages/studyView/table/GenePanelModal.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import { Modal, Button } from "react-bootstrap";
+import { GenePanelToGene, GenePanel } from "shared/api/generated/CBioPortalAPIInternal";
+
+interface IGeneModalProps {
+    genes: GenePanelToGene[];
+    panelName: string;
+    show: boolean;
+    hide: () => void;
+}
+
+interface IGenePanelTooltipProps {
+    genePanels: GenePanel[];
+    toggleModal: (panelName: string, genes: GenePanelToGene[]) => void;
+}
+
+export const GenePanelList: React.FunctionComponent<IGenePanelTooltipProps> = ({
+    genePanels,
+    toggleModal
+}) => {
+    if (genePanels.length > 0) {
+        return (
+            <span>
+                <br />
+                Gene panels:{" "}
+                {genePanels.map((panel, i) => [
+                    i > 0 && ", ",
+                    <a key={panel.genePanelId}
+                        href="#"
+                        onClick={() => {
+                            toggleModal(panel.genePanelId, panel.genes);
+                        }}
+                    >
+                        {panel.genePanelId}
+                    </a>
+                ])}
+            </span>
+        );
+    } else {
+        return <span />;
```

Comment:
null?

Reply:
My thinking here was to avoid a null to simplify the type definition and error checking. Visually this does nothing. I can return null and update the type definitions if you want.